### PR TITLE
Added recipe to install the windows sdk

### PR DESCRIPTION
**Pull request by jdmundrawala**
This will install signtool.exe along with a bunch of other
tools we will probably never need to use.

Also, we need some of @smurawski work around winrm magic. 
I could not do all of this through test-kitchen (winrm).

cc @schisamo @btm @ksubrama @smurawski @mcquin @danielsdeleo

### CODE REVIEW

File: Berksfile [hunk 1]
```diff
@@ -1,6 +1,8 @@
 source 'https://api.berkshelf.com'
 metadata
 
+cookbook 'windows-sdk', github: 'jdmundrawala/windows-sdk-cookbook'
```
Comment (schisamo) [inline Berksfile]: Can we please push a proper release of this cookbook to supermarket.chef.io?

File: metadata.rb [hunk 2]
```diff
@@ -22,3 +22,4 @@
 depends 'homebrew',        '~> 1.9'
 depends 'windows',         '~> 1.30'
 depends 'wix',             '~> 1.1'
+depends 'windows-sdk',     '~> 0.0'
```
Comment (schisamo) [inline metadata.rb]: If this cookbook works just release a 1.0.0 version.

File: Berksfile (hunk 1 above)
Comment (jdmundrawala) [inline Berksfile]: yea, i'll set that up

**Comment by danielsdeleo** [comment]
cool :+1:

**Comment by ksubrama** [comment]
:+1:

**Pull request merged**